src/mdoc/model/DeviceResponse.ts

```diff
@@ -1,3 +1,4 @@
+import { createHash } from 'node:crypto';
```

Comment:
consider using `uncrypto` (this should works on browsers)

Reply:
[Added](https://github.com/auth0-lab/mdl/pull/16/commits/10fe386b4b716b0d63d9e43291673657cbd2752f#diff-99b68b74c11d7bd5d1778b3c48bc8301d2ec023808e88db7c60ac6889676bf38R14-R18) a hash function based on `uncrypto`.
Had to revisit a bit the way we deal with the session transcript, as hashing is done async in `uncrypto` and sync in `node:crypto`